src/test/java/org/openrewrite/staticanalysis/EqualsAvoidsNullTest.java

```diff
@@ -25,604 +25,613 @@
+                  System.out.println(s.compareToIgnoreCase("test"));
               }
-              """
-          )
-        );
+          }
+          """
+      )
+    );
+  }
+
+  @Issue("https://github.com/openrewrite/rewrite-static-analysis/issues/472")
+  @Nested
+  class EqualsAvoidsNullNonIdempotent {
+    @Test
+    void literalAndConstant() {
+      rewriteRun(
+        spec -> spec.recipe(new EqualsAvoidsNull()),
+        // language=java
+        java(
+          """
+            public class Foo {
+                private static final String FOO = "";
+                public void foo() {
+                    FOO.equals("");
+                    "".equals(FOO);
+                }
+            }
+            """,
+          """
+            public class Foo {
+                private static final String FOO = "";
+                public void foo() {
+                    "".equals(FOO);
+                    "".equals(FOO);
+                }
+            }
+            """
+        )
+      );
     }
 
-    @Issue("https://github.com/openrewrite/rewrite-static-analysis/issues/472")
-    @Nested
-    class EqualsAvoidsNullNonIdempotent {
-        @Test
-        void literalAndConstant() {
-            rewriteRun(
-              spec -> spec.recipe(new EqualsAvoidsNull()),
-              // language=java
-              java(
-                """
-                  public class Foo {
-                      private static final String FOO = "";
-                      public void foo() {
-                          FOO.equals("");
-                          "".equals(FOO);
-                      }
-                  }
-                  """,
-                """
-                  public class Foo {
-                      private static final String FOO = "";
-                      public void foo() {
-                          "".equals(FOO);
-                          "".equals(FOO);
-                      }
-                  }
-                  """
-              ));
-        }
-
-        @Test
-        void rawOnRaw() {
-            rewriteRun(
-              //language=java
-              java(
-                """
-                  public class Foo {
-                      public void bar() {
-                          "FOO".equals("BAR");
-                          "FOO".equalsIgnoreCase("BAR");
-                      }
-                  }
-                  """
-              )
-            );
-        }
-
-        @Test
-        void referenceOnReference() {
-            rewriteRun(
-              //language=java
-              java(
-                """
-                  public class Foo {
-                      private static final String FOO = null;
-                      private static final String BAR = null;
-                      public void bar() {
-                          BAR.equals(FOO);
-                      }
-                  }
-                  """
-              )
```

Comment:
we already have 2 spaced indentations mixed with 4. I do not care whats the value, but it should be set and done.

Checkstyle is doing in its test this smaller scope too and google is on this train too. So we might hop on: https://google.github.io/styleguide/javaguide.html#s4.2-block-indentation